diff -Naur portmixer_v18_1/Makefile portmixer_v18_1.oden/Makefile
--- portmixer_v18_1/Makefile	1970-01-01 01:00:00.000000000 +0100
+++ portmixer_v18_1.oden/Makefile	2005-04-10 13:32:28.950263420 +0200
@@ -0,0 +1,27 @@
+# Make PortMixer Library for Unix / OSS
+
+CC = clang
+CFLAGS = $(RPM_OPT_FLAGS) -Ipx_common
+PASRC = px_unix_oss/px_unix_oss.c
+PAINC = px_common/portmixer.h
+PAOBJ = px_unix_oss/px_unix_oss.o 
+LIBS = -lm -lpthread
+AR = /usr/bin/ar
+RANLIB = ranlib
+VERSION=18.1
+
+all: shared static
+
+static: $(PAOBJ) Makefile $(PAINC)
+	$(AR) ruv libportmixer.a $(PAOBJ)
+	$(RANLIB) libportmixer.a
+
+shared: $(PAOBJ) Makefile $(PAINC)
+	$(CC) $(CFLAGS) -shared -Wl,-soname=libportmixer.so.0 -o \
+	libportmixer.so.0.$(VERSION) $(PAOBJ)
+
+clean:
+	rm -f $(PAOBJ) shared static
+
+%.o: %.c $(PAINC) Makefile
+	$(CC) -c $(CFLAGS) $< -o $@
